.github/scripts/tkn_push_bundles.sh

```diff
@@ -14,18 +14,24 @@ printf 'bundle_dirs: %s\n' "${!bundle_dirs[@]}"
 
 # Main loop
 printf 'Pushing any new/modified bundle definitions.\n'
-for BUNDLE_DIR in "${!bundle_dirs[@]}"
+for VERSION_DIR in "${!bundle_dirs[@]}"
```

Comment:
this is confusing to me. If they are version directories, let's rename the array as such. If they array is still holding bundle directories, then let's keep the variable name BUNDLE_DIR as is

Reply:
Good idea. The latest commit changes the variable name to be more clear.